File: Type of array/Type_of_array.java
class Solution {
    int maxNtype(int arr[]) {
        int max = 0;
        int min = 0;
        int length = arr.length;
        
        for(int i = 0; i < length; i++) {
            if(arr[max] < arr[i])
               max = i;
            if(arr[i] < arr[min])
                min = i;  
        }
        
        if(min == 0 && max == lenght-1)
           return 1;
        else if(max == 0 && min == length-1)
           return 2;   
        else if(max > min)
           return 3;
           
        return 4;   
   
    }
}

File: Type of array/Type_of_array.py
class Solution:

	def maxNtype(self,arr):class Solution:

	def maxNtype(self,arr):
        max, min, len = 0, 0, len(arr)

        for i in range(0 ,len):
            if arr[max] < arr[i]:
                max = i
            if arr[min] > arr[i]:
                min = i
            
       if min == 0 and max == len-1:
           return 1
        elif min == len-1 and max == 0:
            return 2
        elif max > min:
            return 3
        
        return 4      
    max, min, len = 0, 0, len(arr)

    for i in range(0 ,len):
        if 
